Lesson 12/src/index.js

```diff
@@ -0,0 +1,108 @@
+// 1
+
+let number = 0
+
+while (number <= 20) {
+  console.log(number)
+  if (number === 15) {
+    break
+  }
+  number += 1
+  // if (a === 10) {
+  //   continue
+  // }
+}
+
+// 2
+function findEvenNumber(number1) {
+  if (number1 % 2 === 0) {
+    return true
+  } return false
```

Comment:
```suggestion
  return number1 % 2 === 0
```

сравнение `number1 % 2 === 0` возвращает true или false, можно сразу и вернуть как результат